roles/nginxplus/vars/main.yml

```diff
@@ -1,2 +1,152 @@
 ---
-# vars file for roles/nginxplus
+# the 'sites' dictionary is used by the library.conf.j2 template
+sites:
+  # - name: example-prod
+    ## 'servers' is a list of VMs that serve this site
+    # servers:
+    #   - example-prod1
+    #   - example-prod2
+    ## short_name is used for cookies for sticky learning
+    # short_name: example
+    ## zone_size is the amount of memory for the cache zone
+    ## default is 64
+    # zone_size: 128
+    ## location defines the URL for the nginx location directive
+    # location: "/"
+    ## the template defaults to restricted configuration (VPN-only)
+    ## set 'visibility: public' to create a site that is visible to the world
+    # visibility: public
+    ## default load balancing technique is round-robin
+    ## see https://www.nginx.com/blog/choosing-nginx-plus-load-balancing-techniques/ for options
+    # lbtechnique: least_time
+    ## most sites have app_protect disabled
+    ## set app_protect: enabled to enable it
+    # app_protect: enabled 
+    ## error handling templates live in /etc/nginx/conf.d/templates/
+    ## the default template for handling errors is errors.conf
+    # error_handling_template: cdh-errors.conf
+    ## the load balancers do health checks at the root of the site
+    ## set health_check_URI to define a different health URI
+    # health_check_URI: /solr
+  - name: abid-prod
+    servers:
+      - abid1
+    short_name: abid
+    location: "/"
+    visibility: public
+    app_protect: disabled
+  - name: abid-staging
+    servers:
+      - abid-staging1
+    short_name: abid-staging
+    location: "/"
+    visibility: private
+    app_protect: enabled
+  - name: allsearch-api-prod
```

Comment:
This might not have been in when you started this, but I think you could do allsearch and allsearch-staging as well (in addition to the api-only boxes)

Reply:
Cool! I'll add them in.